pkg/container/types/types.go

```diff
@@ -81,6 +81,12 @@ const (
 	RowidSize = 16
 )
 
+const (
+	CharNormal int32 = 0
```

Comment:
Is hex a type?

Reply:
Yes, blob is implemented by varchar, so it needs to be distinguished

Reply:
Are hex and blob the same type?

Reply:
Isn't a blob a text with a binary character set.

Reply:
Isn't a blob just a separate type. Isn't T_blob?

Reply:
blob can store data of hex, for example:
create table t1 (a blob);
insert into t1 values(0x01020304FFFFFFFF0000000CAAC0000);
select * from t1;
a
0x01020304FFFFFFFF0000000CAAC0000

Reply:
It seems not correct.
<img width="729" alt="image" src="https://user-images.githubusercontent.com/77312370/194747948-096a7d0f-3b9b-4ce5-9016-6d4890c667ac.png">

Reply:
The type of vector a is varchar which should be blob when read table t1. I am confused by the people who implement this type. The result of show create from t1 is that the type of a is text. The person who implements blob needs to organize.

Reply:
This pr will be closed. This solution will indeed cause some problems